index.html

```diff
@@ -20,7 +20,138 @@ <h1>Charlie<br>Giorgania</h1>
             If you like what you see and have a project you need coded, don't hesitate and contact me.</p> 
             <button type="button" class="rectangle">Start collaboration</button>             
         </section>
-        <div class ="background"></div>
+        <div class ="background img1"></div>
     </div>
+    <section class="grid">
+    <h2 class="padd">Projects</h2>
+    <div class ="background img2"></div>
+        <div class="blue-background">
+            <div class="margin">
+                <h3>Project name goes here</h3>
+                <ul class="flex">
+                    <li class="languages">HTML/CSS</li>
+                    <li class="languages">Ruby on Rails</li>
+                    <li class="languages">JavaScript</li>
+                </ul>
+                <button class="big-button center" type="button">See this project <img src="images/arrow.png" alt="arrow-right"></button>
+            </div>
+        </div>
+    </section>
+    <section class="grid">
+        <div class ="background img3"></div>
+            <div class="blue-background">
+                <div class="margin">
+                    <h3>Project name goes here</h3>
+                    <ul class="flex">
+                        <li class="languages">HTML/CSS</li>
+                        <li class="languages">Ruby on Rails</li>
+                        <li class="languages">JavaScript</li>
+                    </ul>
+                    <div class="break"></div>
+                    <button class="big-button center" type="button">See this project <img src="images/arrow.png" alt="arrow-right"></button>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Kindle add all work card projects inside one `<section>` tag, not each project card in separate `<section>` tag and use display `grid ` to algin the whole section and inside each card align it with `flexbox`.

Reply:
All good now 👍🏽